interface/main/calendar/add_edit_event.php

```diff
@@ -229,16 +234,16 @@ function DOBandEncounter($pc_eid)
                     manage_tracker_status($event_date, $appttime, $temp_eid, $_POST['form_pid'], $_SESSION["authUser"], $_POST['form_apptstatus'], $_POST['form_room'], $encounter);
                 }
             } else {
-                    # Capture the appt status and room number for patient tracker.
+                # Capture the appt status and room number for patient tracker.
                 if (!empty($_GET['eid'])) {
-                    manage_tracker_status($event_date, $appttime, $_GET['eid'], $_POST['form_pid'], $_SESSION["authUser"], $_POST['form_apptstatus'], $_POST['form_room']);
+                    manage_tracker_status($event_date, $appttime, $_GET['eid'], $_POST['form_pid'], $_SESSION["authUser"], $_POST['form_apptstatus'], $_POST['form_room'], $is_tracker);
```

Comment:
the $is_tracker is throwing me off here. Won't this be a boolean when it should be a encounter id?